applications/StructuralMechanicsApplication/custom_response_functions/response_utilities/adjoint_postprocess.cpp

```diff
@@ -202,10 +217,23 @@ namespace Kratos
         std::vector<Vector> sensitivity_vector(num_threads);
         std::vector<Vector> response_gradient(num_threads);
         std::vector<Vector> adjoint_vector(num_threads);
+        std::vector<Vector> adjoint_vector_0(num_threads);
         std::vector<Matrix> sensitivity_matrix(num_threads);
+        
+        double load_factor_ratio;
+
+        // For follower load the ratio is only a double and same for all conditions
+        // This appraoch is a inexpensive approach to calculate the correction term
+        auto condition_pointer = mrModelPart.Conditions().begin();
+        auto cond = *condition_pointer;
+        Matrix residual_gradient;
+        Vector scaling_factor_vector;
+        Vector adjoint_values;
+        mrResponseFunction.CalculateFirstDerivativesGradient(cond, residual_gradient,
```

Comment:
This call makes the adjoint postprocess unusable for other responses, because `CalculateFirstDerivativesGradient`  is not implemented in the other structural responses.

Reply:
I will add a condition so it is only called for non-linear-static mode